### PR TITLE
Class02 과제3 OTX를 이용한 model training - 정용재

### PR DESCRIPTION
**Pull request by eolgaemi**
Class02 과제3 OTX를 이용한 model training - 정용재

### CODE REVIEW

File: class02/homework/yj_jung/hw3/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+./splitted_dataset/val/<Bad>: 12​
+```
+
+## Training 결과
+|Classification model|Accuracy|FPS|Training time|Batch size|Learning rate|Other hyper-prams|
+|----|----|----|----|----|----|----|
+|EfficientNet-V2-S| 1.0| 125| 0:00:08.279238| 64| 0.0071
+|EfficientNet-B0| 1.0| 125| 0:00:08.279238| 64| 0.0049|
+|DeiT-Tiny| 1.0| 125| 0:00:08.279238| 64| 0.0001|
+|MobileNet-V3-large-1x| 1.0| 142|'0:00:07.755650'| 64|0.0058|
```
Comment (JerryJLee) [inline class02/homework/yj_jung/hw3/README.md:22]: 모델 4개를 다 테스트 한 것으로 README가 작성되었는데, 커밋에는 왜 다 누락되었을까요?

Comment (eolgaemi) [reply class02/homework/yj_jung/hw3/README.md:22]: final PR에 같이 올렸습니다!
